renin/src/renin.ts

```diff
@@ -60,6 +60,7 @@ export interface Options {
   productionMode: boolean;
   rendererOptions?: WebGLRendererParameters;
   toneMapping: WebGLRenderer['toneMapping'];
+  aspectRatio: number;
```

Comment:
Does this need a default for demos who haven't set it?

Reply:
Should be default 1?

Reply:
Sat the aspectRatio to 16.0/9 in the example-project instead after a discussion on messenger